tests/mock_data/loom/gene_model_0.txt

```diff
@@ -0,0 +1 @@
+{'name': 'A2M', 'gene_id': None, 'source_file_type': ('loom',), 'cell_names': ['BM01_16dpp_AAGCAGTGGTAT', 'BM01_16dpp_TAAGCAGTGGTA', 'BM01_16dpp_CTAAGCAGTGGT', 'BM01_16dpp_CGGTAAACCATT', 'BM01_16dpp_CCGAATTCACCG'], 'expression_scores': [-0.191, -0.191, 2.646, -0.191, -0.191], 'subdocument': {'cell_names': ['BM01_16dpp_AAGCAGTGGTAT', 'BM01_16dpp_TAAGCAGTGGTA', 'BM01_16dpp_CTAAGCAGTGGT', 'BM01_16dpp_CGGTAAACCATT', 'BM01_16dpp_CCGAATTCACCG'], 'expression_scores': [-0.191, -0.191, 2.646, -0.191, -0.191], 'source_file_type': 'loom'}, 'top_level_doc': {'file_id': '1234abc', 'searchable_name': 'a2m', 'name': 'A2M', 'gene_id': None, 'study_accession': 'SCP1', 'taxon_name': 'Homo Sapiens', 'taxon_common_name': 'human', 'ncbi_taxid': '9606', 'genome_assembly_accession': 'GCA_000001405.15', 'genome_annotation': 'Ensemble 94'}}
```

Comment:
Fixing two minor typos (also updated in test):
* Homo Sapiens -> Homo sapiens
* Ensemble 94 -> Ensembl 94

```suggestion
{'name': 'A2M', 'gene_id': None, 'source_file_type': ('loom',), 'cell_names': ['BM01_16dpp_AAGCAGTGGTAT', 'BM01_16dpp_TAAGCAGTGGTA', 'BM01_16dpp_CTAAGCAGTGGT', 'BM01_16dpp_CGGTAAACCATT', 'BM01_16dpp_CCGAATTCACCG'], 'expression_scores': [-0.191, -0.191, 2.646, -0.191, -0.191], 'subdocument': {'cell_names': ['BM01_16dpp_AAGCAGTGGTAT', 'BM01_16dpp_TAAGCAGTGGTA', 'BM01_16dpp_CTAAGCAGTGGT', 'BM01_16dpp_CGGTAAACCATT', 'BM01_16dpp_CCGAATTCACCG'], 'expression_scores': [-0.191, -0.191, 2.646, -0.191, -0.191], 'source_file_type': 'loom'}, 'top_level_doc': {'file_id': '1234abc', 'searchable_name': 'a2m', 'name': 'A2M', 'gene_id': None, 'study_accession': 'SCP1', 'taxon_name': 'Homo sapiens', 'taxon_common_name': 'human', 'ncbi_taxid': '9606', 'genome_assembly_accession': 'GCA_000001405.15', 'genome_annotation': 'Ensembl 94'}}
```

Reply:
Addressed in f708656